modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -248,6 +254,7 @@ For example, to use DHCP on a NIC that is named `eno1`, set `ip=eno1:dhcp`.
 <3> Specify the location of the `initramfs` file that you uploaded to your TFTP server.
 
 endif::only-pxe[]
+endif::openshift-origin[]
```

Comment:
Same thing with the `endifs`

`endif::openshift-origin[]`         -> `endif::only-pxe,openshift-origins[]` 
`endif::only-pxe[]`